markdown/ch5.markdown

```diff
@@ -1,5 +1,10 @@
 # Contingency tables using sparse coordinate matrices
 
+> I like sparseness. There's something about that minimalist feel that can make
+> something have an immediate impact and make it unique.
+>
+> — Britt Daniel
```

Comment:
I thought about it, but we don't have any for the others... What do you think about "lead singer for Spoon"?